### PR TITLE
default gpt-4, failover

### PR DESCRIPTION
**Pull request by zzstoatzz**

### CODE REVIEW

File: src/marvin/settings.py [hunk 1]
```diff
@@ -44,8 +44,9 @@ def __setattr__(self, name: str, value: Any) -> None:
 
 class ChatCompletionSettings(MarvinSettings):
     model_config = SettingsConfigDict(env_prefix="marvin_chat_completion_")
-    model: str = Field(
-        description="The default chat model to use.", default="gpt-3.5-turbo"
+    model: str = Field(description="The default chat model to use.", default="gpt-4")
+    _fallback_model: str = PrivateAttr(
```
Comment (jlowin) [inline src/marvin/settings.py:48]: I suggest either making this public or moving it to a global variable in the client module; private settings are going to be very difficult to discover.

File: src/marvin/settings.py [hunk 2]
```diff
@@ -44,8 +44,9 @@ def __setattr__(self, name: str, value: Any) -> None:
 
 class ChatCompletionSettings(MarvinSettings):
     model_config = SettingsConfigDict(env_prefix="marvin_chat_completion_")
-    model: str = Field(
-        description="The default chat model to use.", default="gpt-3.5-turbo"
+    model: str = Field(description="The default chat model to use.", default="gpt-4")
```
Comment (jlowin) [inline src/marvin/settings.py:47]:
```suggestion
    model: str = Field(description="The default chat model to use.", default="gpt-4-1106-preview")
```

I think if you have GPT-4 you have the turbo preview, so this is better, cheaper and much larger context

Comment (zzstoatzz) [reply src/marvin/settings.py]: updated in [a657902](https://github.com/PrefectHQ/marvin/pull/746/commits/a657902cacffe49be67e3f20e68f49177d15db84)

File: src/marvin/settings.py (hunk 1 above)
Comment (zzstoatzz) [reply src/marvin/settings.py]: made a var in the client module in [a657902](https://github.com/PrefectHQ/marvin/pull/746/commits/a657902cacffe49be67e3f20e68f49177d15db84)

**Review by jlowin — APPROVED** [review]

**Pull request merged**